app/models/category_localization.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CategoryLocalization < ActiveRecord::Base
```

Comment:
Technically we could be more specific and say translation, but I'm okay with this too. 🙂